src/renderer/lib/validate.ts

```diff
@@ -45,3 +45,21 @@ export const isValidConnectCodeStart = (codeStart: string) => {
 
   return true;
 };
+
+export const isValidDisplayName = (displayName: string): string | true => {
```

Comment:
This is something that I've noticed but I would prefer for us to distinguish between an `isValidX` function (which has the type signature `() => boolean` versus a error validation (which has the type signature `() => string | true`. I think if we return the error message in the function we should call the function name `validateX` and if we simply return a boolean we should call the function name `isValidX`. I think this would make things a bit clearer.

```suggestion
export const validateDisplayName = (displayName: string): string | true => {
```